### PR TITLE
Added prefix in the Peddy command

### PR DESCRIPTION
**Pull request by sima-r**
## PR checklist
Added prefix in the command as it was missed. 
Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/peddy/main.nf [hunk 1]
```diff
@@ -27,6 +27,7 @@ process PEDDY {
     """
     peddy \\
         $args \\
+	--prefix $prefix \\
```
Comment (jemten) [inline modules/peddy/main.nf:30]: align argument

Comment (nvnieuwk) [inline modules/peddy/main.nf:30]:
```suggestion
	    --prefix $prefix \\
```
Indentation fix

**Review by nvnieuwk — COMMENTED** [review]
Can you also fix the stub so the files get the correct prefix?

**Comment by jemten** [comment]
Hej @sima-r, the code looks good but there are some conflicts that needs to be resolved before the tests can run and the PR can be approved.

**Review by jemten — COMMENTED** [review]
I think you've placed the main file in the wrong directory, should be `modules/nf-core/peddy`

**Pull request closed without merging**